src/methods/send.ts

```diff
@@ -73,13 +73,10 @@ const send = async (genomeBrowser: GenomeBrowserType, action: OutgoingAction) =>
     for (const track_id of action.payload.track_ids) {
       genomeBrowser.clear_switch(["track", track_id, "several"])
     }
-  } else if (action.type === OutgoingActionType.TURN_ON_TRANSCRIPTS) {
+  } else if (action.type === OutgoingActionType.SET_VISIBLE_TRANSCRIPTS) {
     const { track_id, transcript_ids } = action.payload;
-    genomeBrowser.set_switch(["track", track_id, "transcript", transcript_ids.join(" ")]);
+    genomeBrowser.set_switch(["track", track_id, "transcripts", transcript_ids.toString()]);
```

Comment:
Not sure I am reading it correctly, but didn't Dan suggest passing an actual array? I.e. `JSON.stringify(transcript_ids)`?